src/NewCommand.php

```diff
@@ -571,27 +572,15 @@ protected function askToInstallAddons()
             return $this;
         }
 
-        $choice = select(
-            label: 'Would you like to install any first-party addons?',
-            options: [
-                $withoutAddonsOption = "No, I'm good for now.",
-                "Yes, let me pick.",
-            ],
-        );
-
-        if ($choice === $withoutAddonsOption) {
-            return $this;
+        if (confirm('Do you plan to generate a static site?', default: false)) {
+            $this->addons[] = 'ssg';
         }
 
-        $this->addons = multiselect(
-            label: 'Which first-party addons do you want to install?',
-            options: [
-                'collaboration' => 'Collaboration',
-                'eloquent-driver' => 'Eloquent Driver',
-                'ssg' => 'Static Site Generator',
-            ],
-            hint: 'Use the space bar to select options.'
-        );
+        if ($this->pro ?? true) {
```

Comment:
```suggestion
        if ($this->pro) {
```

When #82 is merged, we can remove the `?? true`. The Collaboration addon should only be an option when using Statamic Pro.